apps/projects/rules.py

```diff
@@ -7,11 +7,17 @@
 from adhocracy4.projects.predicates import is_moderator
 from adhocracy4.projects.predicates import is_project_member
 from adhocracy4.projects.predicates import is_public
+from adhocracy4.projects.predicates import is_semipublic
 
 rules.add_perm('a4_candy_projects.change_project',
                is_superuser | is_initiator)
 
 rules.add_perm('a4_candy_projects.view_project',
+               is_superuser | is_initiator | is_moderator |
+               ((is_public | is_semipublic | is_org_member | is_project_member)
+                & is_live))
+
+rules.add_perm('a4_candy_projects.participate_in_project',
```

Comment:
I wonder if we should move these to a4, use `a4projects....` rules everywhere and, where adoption is needed, overwrite the `a4projects....` rules using `rules.set_rule()`

Reply:
For the record: that's what we do in mB, so we should most likely do the same here.

Reply:
Done